src/Sentry.Unity.Android/SentryJava.cs

```diff
@@ -51,6 +52,8 @@ internal class SentryJava : ISentryJava
 {
     private readonly IAndroidJNI _androidJNI;
     private IDiagnosticLogger? _logger;
+    private ThreadLocal<bool> DidSdkAttachToJni = new();
```

Comment:
naming: fields should be lowercase and since it's `private ` prefixed with `_`
modifier: can be `readonly`